src/gui/properties/propertieswidget.ui

```diff
@@ -83,7 +83,7 @@
                </widget>
               </item>
               <item row="0" column="2">
-               <widget class="QLabel" name="progress_lbl">
+               <widget class="QLabel" name="progress">
```

Comment:
I didn't add a "label" prefix or suffix because these are used in C++ but sure i'll add it as suffix and also change the prefix to suffix on existing ones.

Reply:
IMO, prefix is better choice: from Designer you usually know the type of a given widget and can start typing the prefix to get a list of choices from a completion system.

Reply:
Good point, prefix then.

Reply:
Agree.